.github/ISSUE_TEMPLATE/2_documentation.yml

```diff
@@ -0,0 +1,33 @@
+name: Documentation
+description: Create a documentation related issue.
+labels:
+  - type::documentation
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > **Note**
+        Since there are already a lot of open issues, please also take a moment to search existing ones to see if your bug has already been reported. If you find something related, please upvote that issue and provide additional details as necessary.
+        💐 Thank you for helping to make Conda better. We would be unable to improve Conda without our community!
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: Please confirm and check all of the following options.
+      options:
+        - label: I added a descriptive title
+          required: true
+        - label: I searched open reports and couldn't find a duplicate
+          required: true
+  - type: textarea
+    id: what
+    attributes:
+      label: What happened?
+      description: Mention here any typos, broken links, missing, incomplete, or outdated information etc. that you have spotted in the conda docs or CLI help.
```

Comment:
```suggestion
      description: Mention here any typos, broken links, or missing, incomplete, or outdated information, etc. that you have noticed in the conda docs or CLI help.
```

Reply:
Replacing "spotted" with "noticed", because "spotted" feels like it would translate weirdly. Not totally sure about that, as I only speak English. Whomp whomp.